components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/dao/CacheBackedLocalClaimDAO.java

```diff
@@ -38,6 +40,7 @@ public class CacheBackedLocalClaimDAO {
     LocalClaimDAO localClaimDAO;
 
     LocalClaimCache localClaimInvalidationCache = LocalClaimCache.getInstance();
+    AssociatedClaimCache associatedClaimCache = AssociatedClaimCache.getInstance();
```

Comment:
Can't see the logic for cache invalidation when adding new external claim mapping for the local claim URI/ deleting an external claim mapping